.chloggen/breaking-3996.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: breaking
+
+# The name of the component, or a single word describing the area of concern, (e.g. collector, target allocator, auto-instrumentation, opamp, github action)
+component: otel-allocator
+
+# A brief description of the change. Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Remove overwrite of global config SraperProtocols, by default now it will include `PrometheusText1.0.0` that requires prometheusreceiver >0.120.0
```

Comment:
```suggestion
note: Remove overwrite of global config ScraperProtocols, by default now it will include `PrometheusText1.0.0` that requires prometheusreceiver >0.120.0
```

Reply:
Done.